### PR TITLE
Fix pageviews flood on route transitions

### PR DESCRIPTION
**Pull request by phndiaye**
Setup the google analytics script using an instance-initializer (which has direct access to the app's service injection system compared to initializers). Doing so allows us to avoid reopening the `Route` class.

### CODE REVIEW

**Review by Miexil — APPROVED** [review]

File: addon/instance-initializers/g-analytics-init.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      if (transition.intent && transition.intent.name && window.ga) {
+        const sendPageView = () => {
+          window.ga(
+            "send",
+            "pageview",
+            this.router.urlFor(transition.intent.name)
+          );
+        };
+
+        sendPageView();
```
Comment (JulienVannier66) [inline addon/instance-initializers/g-analytics-init.js]:
```suggestion
        window.ga(
          "send",
          "pageview",
          this.router.urlFor(transition.intent.name)
        );
```
No working with the arrow function?

Comment (phndiaye) [reply addon/instance-initializers/g-analytics-init.js]: updated :p i tried something then forgot to move it back

Comment (JulienVannier66) [reply addon/instance-initializers/g-analytics-init.js]: You update that in your last commit, thanks github ^^

**Review by JulienVannier66 — APPROVED** [review]
LGTM

**Pull request merged**